src/Base.ts

```diff
@@ -317,7 +317,7 @@ export class Base {
       }
 
       if (typeof v === 'bigint') {
-        return Buffer.from(v.toString(16), 'hex')
+        return Buffer.from(v.toString(16).length % 2 ? '0' + v.toString(16) : v.toString(16), 'hex')
```

Comment:
Context for Buffer lib implementation: https://github.com/nodejs/node/issues/21242